test_scripts/API/Restructuring_OnResetTimeout/007_GENERIC_ERROR_in_case_HMI_did_not_respond.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 6000) to SDL
+-- 4) HMI does not send response.
+-- SDL does:
+-- 1) Respond with GENERIC_ERROR resultCode to mobile app.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local paramsForRespFunction = {
+	notificationTime = 6000,
+	resetPeriod = 6000
+}
+
+local rpcResponse = { success = false, resultCode = "GENERIC_ERROR" }
```

Comment:
rpcResponse is used in another scripts too, so create variable in common file and use it from common.